testing/jormungandr-integration-tests/src/mock/mod.rs

```diff
@@ -1,20 +1,14 @@
 extern crate base64;
 extern crate futures;
-extern crate futures_cpupool;
-extern crate grpc;
 extern crate hex;
-extern crate protobuf;
 
```

Comment:
You can remove all of these, it's 2018 edition now. The crate imports with `[macro_use]` can stay for now.